### PR TITLE
Add migration docs from <v2 to >=v2

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/sdk/storage-v2.mdx [hunk 1]
```diff
@@ -360,9 +360,9 @@ interface Upload {
 Used to migrate the Buckets and their Uploads from Static Site organization to Storage organization.
 
 <Callout type="info">
-  **NOTE:** Important: To execute the migration you need to use the token of the
-  Storage organization, and you need to be the owner of both the Storage and the
-  Static Site organizations. it.
+  To execute the migration you need to use the token of the Storage
+  organization, and you need to be the owner of both the Storage and the Static
+  Site organizations. it.
```
Comment (rekpero) [inline pages/sdk/storage-v2.mdx:365]: what is `it.` at the end doing?

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/migration.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Callout } from "nextra-theme-docs";
+
+# Migration from &lt;2.0.0 to >=2.0.0
```
Comment (rekpero) [inline pages/storage/migration.mdx:3]: name the sidebar title `SDK Migration`

Comment (rekpero) [inline pages/storage/migration.mdx:3]: Also 
```suggestion
# SDK Migration from &lt;2.0.0 to >=2.0.0
```

File: pages/storage/migration.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Main Changes
+
+Version **2.0.0** of the `@spheron/storage` SDK brings significant changes and improvements.
+Here are the main changes you need to be aware of when migrating from versions earlier than 2.0.0 to 2.0.0 or later:
+
+**1. Token Requirement**
+
+- In versions prior to 2.0.0, the SDK required a token for a Static Site organization.
+- Starting from version 2.0.0, the SDK now requires a token for a **Storage** organization. All SDK usages are bound to this Storage organization.
```
Comment (rekpero) [inline pages/storage/migration.mdx:13]: mention how to create token for current SDK

File: pages/storage/migration.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+**3. IPNS Records**
+
+- IPNS Records are now grouped at the bucket level. Please refer to the relevant section for more detailed information.
+
+## Changes Required to Migrate
+
+To migrate from a version of the SDK prior to 2.0.0 to version 2.0.0 or later, follow these steps:
+
+1. Make sure you are the owner of both the Static Site organization and the new Storage organization.
+2. Create an access token for the **Storage** organization. This token should be used when initializing the **SpheronClient**.
```
Comment (rekpero) [inline pages/storage/migration.mdx:34]: Add this `Learn how to create access token here`

File: pages/storage/migration.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import { Callout } from "nextra-theme-docs";
+
+# Migration from &lt;2.0.0 to >=2.0.0
+
+## Main Changes
+
+Version **2.0.0** of the `@spheron/storage` SDK brings significant changes and improvements.
```
Comment (rekpero) [inline pages/storage/migration.mdx:7]: @spheron/storage should be link to the npm package

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comments

File: pages/storage/migration.mdx (hunk 2 above)
Comment (aayushmahapatra) [reply pages/storage/migration.mdx:3]: Done in #102

File: pages/storage/migration.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+<Callout type="warning">
+  After you migrate the data it is not possible to revert it.
+</Callout>
+
+```js
+const { numberOfBuckets, numberOfUploads } =
+  await client.migrateStaticSiteOrgToStorage(
+    storageOrganizationId,
+    staticSiteOrganizationId
```
[the remark this answers is not in the document]
Comment (aayushmahapatra) [reply pages/storage/migration.mdx:52]: It's correct. Confirmed with @shomix .

File: pages/storage/migration.mdx (hunk 3 above)
Comment (aayushmahapatra) [reply pages/storage/migration.mdx:13]: Done.

**Review by rekpero — APPROVED** [review]

**Pull request merged**